src/terrain/routes/token.clj

```diff
@@ -1,9 +1,45 @@
 (ns terrain.routes.token
-  (:use [compojure.core :only [routes GET]]
-        [terrain.services.oauth :only [get-token]]))
+  (:use [compojure.api.core :only [context routes GET]]
+        [terrain.routes.schemas.token]
+        [terrain.services.oauth :only [get-token]])
+  (:require [clojure.string :as string]))
+
+(def ^:private token-route-description
```

Comment:
Yeah, I struggled with this myself. I generally try to avoid anything that messes with the indentation, and big multiline strings certainly do that. I also thought about putting the strings in a separate file, but decided not to because of all of the hoops that we'd have to jump through. I kind of wish that Clojure had the equivalent of a heredoc. That also messes up the indentation, but at least editors are equipped to deal with it in languages that have them.

Using `string/join` seemed like a reasonable compromise, but it does make editing the description somewhat painful. It might actually be worthwhile to have code that slurps in the files. It might require some extra work, but once we have the framework in place, it should be easy to maintain.

Reply:
Hmm, yeah, there might be some good templating options. I'm not sure what values we'd actually plug in to the templates, though. We could add some custom tags to the endpoint definitions so that we can define our own template values. I'll try to find ways of doing that cleanly. If I can find something, I'll switch over to using that in the next PR.

Reply:
Yeah -- for this example we don't need the templating, really. I was thinking of it more for documenting "this endpoint can return these error codes", where we'd pass in a list (whether an actual list or an already-assembled string list) of the codes. Or similarly, if it's an endpoint that delegates to another service, we could probably write a way to template that out (data-info has `get-endpoint-delegate-block` which does this one). That's a different need than just handling this particular long string, though -- I was just thinking that if we set up templating, maybe it can end up solving both problems.

Reply:
I might be saying things you already understood from my past response though, now that I think about it. Anyway, your plan sounds good to me!

Reply:
> Using `string/join` seemed like a reasonable compromise, but it does make editing the description somewhat painful. It might actually be worthwhile to have code that slurps in the files. It might require some extra work, but once we have the framework in place, it should be easy to maintain.

I like this `string/join` solution over the big multi-line strings we have in our other swagger docs.

I also think slurping in a markdown file would be the best solution, ultimately. Not only would it allow us to use an editor that can preview the markdown, but then docs that should be shared between services (such as some of the terrain passthrough endpoints) can have their markdown docs stored in the `common-swagger-api` lib, perhaps.